job-server/src/main/scala/spark/jobserver/JobManagerActor.scala

```diff
@@ -230,7 +230,7 @@ class JobManagerActor(contextConfig: Config, daoActor: ActorRef)
       None
     }
 
-    val daoAskTimeout = Timeout(3 seconds)
+    val daoAskTimeout = Timeout(600 seconds)
```

Comment:
I don't think we should change this. Things should timeout normally if something goes wrong